201_numerik_mit_numpy.ipynb

```diff
@@ -1075,7 +1038,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#np.loadtxt?"
+    "# np.loadtxt?"
```

Comment:
Hier fehlen Erklärungen wofür diese Zellen gut sind

Reply:
Doch, das müsste eig im Text drüber erklärt werden!

Reply:
Ich glaube du hast mich falsch verstanden. Ja, oben wird erklärt was loadtxt und savetxt tun, aber nicht, warum hier eine Zeile mit nur einem `# np.loadtxt?` drin, ist.

Reply:
also ich glaub, ich hab dich tatsächlich nicht verstanden. Diese Zeile müsste eigentlich so gerendert werden: 
![grafik](https://github.com/timomit/dsa-python-course/assets/19753621/b323a2c1-db81-4d68-8c74-93fa9cd6b9da)

Und da steht darüber für mich klipp und klar, was zu tun ist!

Reply:
https://github.com/timomit/dsa-python-course/pull/7#discussion_r1224062103 Meine Antwort ist iwie nach unten gerutscht!

Reply:
wow. ich habe 3 mal nach sowas in der richtung gesucht und das hier nicht gesehen...Kannst du das (uneingerückt) direkt über die Zelle packen? Also, it might be me, aber wenn man es sucht und trotzdem mehrfach übersieht...